lists/list.css

```diff
@@ -0,0 +1,27 @@
+.list {
+  max-width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+ul{
+  margin:0;
+}
+
+.no-bullet-list > ul {
+  list-style-type: none;
+  margin: 0;
```

Comment:
Note that when you set an attribute for `ul` above, it'll apply to all `ul`s in the DOM. Be careful about specificity rules.

Reply:
yes...i'll keep tht in mind